extensions/amp-lightbox/1.0/base-element.js

```diff
@@ -54,8 +54,8 @@ BaseElement['Component'] = Lightbox;
 
 /** @override */
 BaseElement['props'] = {
-  'animation': {attr: 'animation'},
-  'scrollable': {attr: 'scrollable', type: 'boolean'},
+  'animation': {attr: 'animation', media: true},
+  'scrollable': {attr: 'scrollable', type: 'boolean', media: true},
```

Comment:
`animation` makes perfect sense. But `scrollable` not really. It's actually likely cause mistakes since it's very hard to predict when a media condition would make something scrollable or not.